src/wallet/rpcwallet.cpp

```diff
@@ -4915,7 +4915,7 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             } else {
                 throw JSONRPCError(
                     RPC_INVALID_PARAMETER,
-                    "Unified address contained no recognized receiver types.");
+                    "Unified address contained no recognized receiver types, or contained only receiver types for which the required consensus feature is not yet active.");
```

Comment:
```suggestion
                    "Unified address contained only receiver types that are unrecognised or for which the required consensus feature is not yet active.");
```
(It could contain both unrecognised and recognized-but-not-yet-active receiver types.)